sdk/src/feature_set.rs

```diff
@@ -841,6 +841,10 @@ pub mod vote_only_retransmitter_signed_fec_sets {
     solana_sdk::declare_id!("RfEcA95xnhuwooVAhUUksEJLZBF7xKCLuqrJoqk4Zph");
 }
 
+pub mod enable_transaction_failure_fees {
```

Comment:
can we call this `enable_transaction_loading_failure_fees`?
Otherwise this sounds like previously we did not allow any transaction failures